core/modules/layout/layout.module

```diff
@@ -90,6 +92,105 @@ function layout_menu() {
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+  );
+
+  $items['admin/structure/layouts/settings/flexible-template/%/configure'] = array(
+    'title' => 'Configure flexible layout template',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('layout_flexible_template_configure_form', 5),
+    'type' => MENU_NORMAL_ITEM,
+    'weight' => -10,
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+    'theme callback' => 'ajax_base_page_theme',
+  );
+
+  $items['admin/structure/layouts/settings/flexible-template/%/delete'] = array(
+    'title' => 'Delete flexible layout template',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('layout_flexible_template_delete_form', 5),
+    'type' => MENU_NORMAL_ITEM,
+    'weight' => -10,
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+    'theme callback' => 'ajax_base_page_theme',
+  );
+
+  // Put all code in separate file to make dev easier, because layout.admin is huge.
+  $items['admin/structure/layouts/settings/flexible-template/%/row/%/delete'] = array(
+    'title' => 'Delete row',
+    'page callback' => 'layout_flexible_template_delete_row',
+    'page arguments' => array(5,7),
+    'type' => MENU_CALLBACK,
+    'weight' => -10,
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+    'theme callback' => 'ajax_base_page_theme',
+  );
+
+  $items['admin/structure/layouts/settings/flexible-template/%/row/%/select-widths'] = array(
+    'title' => 'Choose region widths',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('layout_flexible_template_region_style_select', 5, 7),
+    'type' => MENU_CALLBACK,
+    'weight' => -10,
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+    'theme callback' => 'ajax_base_page_theme',
+  );
+
+  $items['admin/structure/layouts/settings/flexible-template/%/row/%/configure'] = array(
+    'title' => 'Add row',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('layout_flexible_template_edit_row_form', 5, 7),
+    'type' => MENU_CALLBACK,
+    'weight' => -10,
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+    'theme callback' => 'ajax_base_page_theme',
+  );
+
+  $items['admin/structure/layouts/settings/flexible-template/%/row/%/configure/%'] = array(
+    'title' => 'Add row',
+    'page callback' => 'backdrop_get_form',
+    'page arguments' => array('layout_flexible_template_edit_row_form', 5, 7, 9),
+    'type' => MENU_CALLBACK,
+    'weight' => -10,
+    'access arguments' => array('administer layouts'),
+    'file' => 'layout.flexible.inc',
+    'theme callback' => 'ajax_base_page_theme',
+  );
+
+  $items['admin/structure/layouts/settings/flexible-template-enable/%/%'] = array(
```

Comment:
This seems fine but I would suggest making two menu entries at the same `/flexible-template/` path.

```
   $items['admin/structure/layouts/settings/flexible-template/%/enable'] = array(
...
   $items['admin/structure/layouts/settings/flexible-template/%/disable'] = array(
```

Reply:
Done